src/pages/steps/StepsForm.tsx

```diff
@@ -266,7 +274,10 @@ const StepsForm = ({
             {footerStatus === FooterStatus.PUBLISH && [
               <Button
                 variant={ButtonVariants.SUCCESS}
-                onClick={async () => publishOffer()}
+                onClick={async () => {
+                  triggerHotjarSurvey();
+                  publishOffer();
+                }}
```

Comment:
```suggestion
                onClick={() => {
                  triggerHotjarSurvey();
                  publishOffer();
                }}
```

The function awaits nothing